CHANGES.rst

```diff
@@ -12,17 +12,47 @@ The current role maintainer_ is ypid_.
 
 
 debops.apache v0.1.0 - unreleased
-----------------------------------------
+---------------------------------
 
 Added
 ~~~~~
 
 - Initial coding and design. [ypid_]
 
 - Add/Set the default `Referrer Policy`_ to ``no-referrer`` and made it
-  configurable via :ref:`item.http_referrer_policy <apache__ref_vhosts_http_referrer_policy>`.
+  configurable via :ref:`item.http_referrer_policy <apache__ref_vhost_http_referrer_policy>`.
   [ypid_]
 
+- Add the :envvar:`apache__mpm_max_connections_per_child` variable to allow to
+  configure the number of requests a child process should handle before
+  terminating. [ypid_]
+
+- Add support to enable and configure `Apache mod_status`_. You can set
+  :envvar:`apache__status_enabled` to ``True`` to enable it and make the status
+  page accessible from localhost. [ypid_]
+
+- Add support for generic Apache template generation using the `Apache IfVersion directive`_.
+  This feature can be configured by :envvar:`apache__config_use_if_version` and
+  :envvar:`apache__config_min_version`. [ypid_]
+
+- Add Ansible tags for env roles. To only prepare the Apache role
+  environment, you can use the ``role::apache:env`` tag. [ypid_]
+
+Changed
+~~~~~~~
+
+- Change default virtual host server name from ``000-default`` to ``default.{{
```

Comment:
How much does debops care about users who upgrade this role? E.g. should it be mentioned what the user has to do to get rid of old, unmaintained files?  In this case, it seems that 000-default needs to be deleted manually.